### PR TITLE
[Consensus] Add tracing for proposal receiving

### PR DESCRIPTION
**Pull request by sitalkedia**
### Description

This will help figure out if certain channels/threads in consensus is the bottleneck or just sending the proposal over the wire takes time. 

### Test Plan

Existing UT

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4160)

### CODE REVIEW

File: consensus/src/epoch_manager.rs [hunk 1]
```diff
@@ -784,6 +785,12 @@ impl EpochManager {
         peer_id: AccountAddress,
         event: VerifiedEvent,
     ) -> anyhow::Result<()> {
+        if let VerifiedEvent::ProposalMsg(proposal) = &event {
+            observe_block(
+                proposal.proposal().timestamp_usecs(),
+                BlockStage::EPOCH_MANAGER_RECEIVED,
```
Comment (zekun000) [inline consensus/src/epoch_manager.rs:791]: this is not exactly when it's received, it's already after verification (which takes 10+ms for large block I think), we should probably separate them

File: consensus/src/block_storage/tracing.rs [hunk 2]
```diff
@@ -9,6 +9,8 @@ pub struct BlockStage;
 
 impl BlockStage {
     pub const SIGNED: &'static str = "signed";
+    pub const NETWORK_RECEIVED: &'static str = "network_received";
+    pub const EPOCH_MANAGER_RECEIVED: &'static str = "epoch_manager_received";
     pub const RECEIVED: &'static str = "received";
```
Comment (zekun000) [inline consensus/src/block_storage/tracing.rs:14]: we should probably change it to round manager received

File: aptos-move/framework/cached-packages/src/aptos_token_sdk_builder.rs [hunk 3]
```diff
@@ -475,8 +475,8 @@ pub fn token_mint_script(
 }
 
 /// mutate the token property and save the new property in TokenStore
-/// if the token property_version is 0, we will create a new property_version per token and store the properties
-/// if the token property_version is not 0, we will just update the propertyMap
+/// if the token property_version is 0, we will create a new property_version per token to generate a new token_id per token
```
Comment (movekevin) [inline aptos-move/framework/cached-packages/src/aptos_token_sdk_builder.rs:478]: Where did these changes come from?

Comment (zekun000) [reply aptos-move/framework/cached-packages/src/aptos_token_sdk_builder.rs:478]: I think it's rebuild for whatever reason, it didn't get updated for the pr introduced the change

File: consensus/src/epoch_manager.rs (hunk 1 above)
Comment (sitalkedia) [reply consensus/src/epoch_manager.rs]: Done.

**Review by zekun000 — APPROVED** [review]

**Review by msmouse — APPROVED** [review]

**Pull request merged**